requirements.txt

```diff
@@ -0,0 +1,5 @@
+cortx-hare
+cortx-motr
```

Comment:
cortx-motr- Do we need this ? when does mini provisioner call motr ? If its indirect dependency then we can remove

Reply:
Not needed, removed.